test/mail_tls_dhparams.bats

```diff
@@ -0,0 +1,139 @@
+load 'test_helper/common'
+
+# Test case
+# ---------
+# By default, this image is using audited FFDHE groups (https://github.com/docker-mailserver/docker-mailserver/pull/1463)
+#
+# This test case covers the described case against both boolean states for `ONE_DIR`.
+#
+# Description:
+# 1. Verify that the file `ffdhe4096.pem` has not been modified (checksum verification).
+# 2. Verify Postfix and Dovecot are using the default `ffdhe4096.pem` from Dockerfile build.
+# 3. When custom DHE parameters are supplied by the user as `/tmp/docker-mailserver/dhparams.pem`:
+#    - Verify Postfix and Dovecot use the custom `custom-dhe-params.pem` (contents is actually `ffdhe2048.pem`).
+#    - A warning is raised about usage of potentially insecure parameters.
+
+function setup() {
+    run_setup_file_if_necessary
+}
+
+function teardown() {
+    docker rm -f mail_dhparams
+    run_teardown_file_if_necessary
+}
+
+function setup_file() {
+  # Delegated container setup to common_container_setup
+  # DRY - Explicit config changes between tests are more apparent this way.
+
+  # Global scope
+  # Copies all of `./test/config/` to specific directory for testing
+  # `${PRIVATE_CONFIG}` becomes `$(pwd)/test/duplicate_configs/<bats test filename>`
+  export PRIVATE_CONFIG
+
+  export DMS_ONE_DIR=1 # default
+
+  local DH_DEFAULT_PARAMS
+  export DH_DEFAULT_CHECKSUM
+  export DH_CUSTOM_PARAMS
+  export DH_CUSTOM_CHECKSUM
+
+
+  DH_DEFAULT_PARAMS="$(pwd)/target/shared/ffdhe4096.pem"
+  DH_DEFAULT_CHECKSUM="$(sha512sum ${DH_DEFAULT_PARAMS} | awk '{print $1}')"
```

Comment:
The first linting actually makes sense issue:) Here is the change you'd need to do:

``` SUGGESTION
  DH_DEFAULT_CHECKSUM="$(sha512sum "${DH_DEFAULT_PARAMS}" | awk '{print $1}')"
```

Reply:
Ah ok, I wasn't sure if that was the right way to handle it, as I thought that became `"quoted"unquoted"quoted"` which seemed like what the linter tries to avoid happening with vars?